### PR TITLE
devise

### PR DESCRIPTION
**Pull request by varghes1**
I did rake db:create:all and rake db:migrate to set it up.

I installed the devise gem for the users. I also created a static_pages so as to have something in the routes.rb

I still need to input the heroku link for the production once that is complete

### CODE REVIEW

File: app/controllers/static_pages_controller.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class StaticPagesController < ApplicationController
+
+	def index
```
Comment (KaoruDev) [inline app/controllers/static_pages_controller.rb]: sorry can you fix your indentation to 2 spaces?

Comment (KaoruDev) [inline app/controllers/static_pages_controller.rb]: unless you're planning on destroying this controller, which in case nm

**Comment by KaoruDev** [comment]
> I still need to input the heroku link for the production once that is complete
> Not sure what you mean by that ^
> In the devise instructions - in order to deploy to heroku, i need to add the heroku link in 'In production, :host should be set to the actual host of your application.'

File: app/controllers/static_pages_controller.rb (hunk 1 above)
Comment (varghes1) [inline app/controllers/static_pages_controller.rb]: I usually just do a tab, but i fixed it to two spaces

**Comment by varghes1** [comment]
I changed indentations and added comments

**Comment by jmundraw** [comment]
This looks good to merge

**Pull request merged**